docs/manual/troubleshooting.tex

```diff
@@ -838,6 +854,22 @@ \subsection{Build the Type Annotations compiler\label{building-compiler}}
 %END LATEX
 
 \item
+{\bf [Mac OSX 10.5 or later]} Similarly to Ubuntu, add one of the following commands in your \code{.bash\_profile} or \code{.profile} file.
```

Comment:
Drop "Simillarly to Ubuntu".  Mac users shouldn't have even read the Ubuntu instructions, so this is just confusing and might make them read more than they need to.